pkg/generate/app/pipeline.go

```diff
@@ -479,7 +479,8 @@ func (a *acceptBuildConfigs) Accept(from interface{}) bool {
 	if err != nil {
 		return false
 	}
-	return gvk[0].GroupKind() == build.Kind("BuildConfig") || gvk[0].GroupKind() == image.Kind("ImageStream")
+	return gvk[0].GroupKind() == build.Kind("BuildConfig") || gvk[0].GroupKind() == image.Kind("ImageStream") ||
```

Comment:
done, I also added IsResourceOrLegacy() for the same reason (we did that in dozen of places)